src/lib/types/restrictions.ts

```diff
@@ -109,6 +109,7 @@ const fixedOutputMetadataValueType = z.object({
   metadataValueType: metadataValueType,
 })
 
+// leaving as it's being used in other file.
```

Comment:
Is there a reason they're different though?

Reply:
no, it was just to keep an old one for sake of reference